cypress/e2e/shared/firstmile.tests.ts

```diff
@@ -7,7 +7,6 @@ describe('First mile experience', () => {
 
   it('should load the main home-page that has the language tiles', function() {
     cy.getByTestID('home-page--header').should('exist')
```

Comment:
we will need to add `cy.getByTestID('language-tiles--scrollbox').should('exist')` back. Since it applies to the first mile flow, it makes sense to include it in the test.

Reply:
@ChitlangeSahas where is this test id coming from? David had to remove it because the test fails locally because cypress can't find it

Reply:
from what I remember this was for :

<img width="1138" alt="Screen Shot 2022-07-29 at 8 52 27 AM" src="https://user-images.githubusercontent.com/18511823/181797416-570ad103-2996-4cfe-a264-b750d7519896.png">

does it give you anything if you global search for `home-page--header`?

Reply:
ooh sorry i mean `language-tiles--scrollbox` <- this test id